### PR TITLE
Issue #3044358: Update travis to PHP 7.3

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

**Comment by chrfritsch** [comment]
Mystery solved:

```
handleUpload uses coninue in a switch: https://cgit.drupalcode.org/dropzonejs/tree/src/UploadHandler.php
Triggering a PHP error:

PHP 7.3 Bug: "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"

```

**Review by alexpott — DISMISSED** [review]
Looks good. I wonder when we're going to drop PHP 5 support. There is update support for PHP5 in 8.7.x but you can't install on PHP 5 (for what it is worth).

File: docs/development.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Documentation how to check your code for coding style issues can be found [here]
 
 ## Thunder Travis CI
 
-All Thunder pull requests are execute on [Travis CI](https://travis-ci.org/BurdaMagazinOrg/thunder-distribution). On every pull request tests will be executed (or when new commits are pushed into pull request branch). Tests are executed against PHP versions 5.6 (with drush make install) and 7.2 (with composer install). All code will be checked against coding style.
+All Thunder pull requests are execute on [Travis CI](https://travis-ci.org/BurdaMagazinOrg/thunder-distribution). On every pull request tests will be executed (or when new commits are pushed into pull request branch). Tests are executed against PHP versions 5.6 (with drush make install) and 7.3 (with composer install). All code will be checked against coding style.
```
Comment (dbosen) [inline docs/development.md]: This comment is not correct anymore, we do not test against 5.6 for a while now.

**Review by dbosen — CHANGES_REQUESTED** [review]

**Comment by dbosen** [comment]
> 
> 
> Looks good. I wonder when we're going to drop PHP 5 support. There is update support for PHP5 in 8.7.x but you can't install on PHP 5 (for what it is worth

**Review by dbosen — APPROVED** [review]

**Pull request merged**